keyboards/niu_mini/keymaps/planck/rules.mk

```diff
@@ -0,0 +1 @@
+AUDIO_ENABLE = yes
```

Comment:
is this set here just in case someone manages to solder a speaker in?

Reply:
Ya, this keymap actually has audio keycodes in it. So it kinda made sense to keep it enabled in this case.

Reply:
Actually I should remove that as the firmware is well over size.

Reply:
Removed